milatools/datasets/split.py

```diff
@@ -0,0 +1,64 @@
+from dataclasses import dataclass
+
+import numpy
+
+
+@dataclass
+class Split:
+    """Returns 3 Splits of the main data set. The splits are indices of the samples inside the data set"""
+
+    train: numpy.array
+    valid: numpy.array
+    test: numpy.array
+
+    def items(self):
+        return ("train", self.train), ("valid", self.valid), ("test", self.test)
+
+    def __getitem__(self, item):
+        return getattr(self, item)
+
+    def __setitem__(self, key, value):
+        return setattr(self, key, value)
+
+    def keys(self):
+        return "train", "valid", "test"
+
+    def values(self):
+        return self.train, self.valid, self.test
+
+
+def original_split(
+    datasets, data_size=None, seed=None, ratio=None, index=None, balanced=None
```

Comment:
All the parameters seem unused?

Reply:
I did not yet put the [other split methods](https://github.com/mila-iqia/olympus/tree/master/olympus/datasets/split) but there will be used

Reply:
I can't see that repo :cry:

Reply:
I added one of the split as an example